installers/uninstall-osx.bash

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -e
+
+# This function writes the cloud watch configuration file details for either AWS Cognito credentials (Y) or AWS CLI credentials (N)
+function uninstall_answer () {
+
+read -p "
+Are you sure you want to uninstall CloudWatch Publisher?: (Y/N)(Don't know what this is? Press 'n' then enter) " unin_ans
+
+if [ -z "$unin_ans" ]; then
```

Comment:
This seems excessive. I'd say anything other than Y or y can be treated as no and you can exit.

Reply:
will simplify.